ci/docker_build_and_fuzzit.sh

```diff
@@ -40,11 +41,11 @@ cp ./relwithdebinfo/arrow-ipc-fuzzing-test ./relwithdebinfo/out/fuzzer
 ldd ./relwithdebinfo/arrow-ipc-fuzzing-test | grep "=> /" | awk '{print $3}' | xargs -I '{}' cp -v '{}' ./relwithdebinfo/out/.
 cd ./relwithdebinfo/out/
 tar -czvf fuzzer.tar.gz *
+stat fuzzer.tar.gz
 cd ../../
 
 export TARGET_ID=u79f6bXYgNH4NkU99iWK
 export FUZZIT_API_KEY=${FUZZIT_API_KEY:-ac6089a1bc2313679f2d99bb80553162c380676bff3f094de826b16229e28184a8084b86f52c95112bde6b3dbb07b9b7}
```

Comment:
this is also no longer necessary. go to https://app.fuzzit.dev/orgs/apache-arrow/settings and copy the new API Key to crossbow environment variables (I don't have access to this)

Reply:
But we'd like the docker-compose setup to be runnable from a local machine, I think.

Reply:
so there are two workflows fuzzing and regression.

fuzzing usually should only run from the CI when you push new code to master as this update the fuzzer on Fuzzit servers.

Regression is currently not enabled as far as I can see in this project. to enable regression you need to run the same `./fuzzit create job ....` but with `--type local-regression`. this you can run from local machine as well and you don't need an api key for that at all.

To setup the regression in the CI you want to run it on every pull-request or even every commit. the fuzzing workflow should be called only on new pushed to master. I'm not sure how to do it with crossbow but I guess it shouldn't be too complicated.

(Hope it wasn't too confusing:))

Reply:
@kszucs How would we create an environment variable in crossbow?

Reply:
I think it would be better to have a separate docker-compose entries for the two fuzzit jobs.

```yaml
  fuzzit:
    image: arrow:fuzzit
    build:
      context: .
      dockerfile: dev/fuzzit/Dockerfile
    environment:
      CI_ARROW_SHA: ${CI_ARROW_SHA}
      CI_ARROW_BRANCH: ${CI_ARROW_BRANCH}
    volumes: *ubuntu-volumes

  fuzzit-regression:
    image: arrow:fuzzit
    build:
      context: .
      dockerfile: dev/fuzzit/Dockerfile
    environment:
      CI_ARROW_SHA: ${CI_ARROW_SHA}
      CI_ARROW_BRANCH: ${CI_ARROW_BRANCH}
      # arbitrary env variables
    volumes: *ubuntu-volumes
```

Then we can run `docker-compose run fuzzit` and `docker-compose run fuzzit-regression` locally. After that we can setup specific crossbow tasks to run either one or both of them.